Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+\section{Processi di supporto}
+\subsection{Processo di documentazione}
+\subsubsection{Scopo}
+L'obiettivo proposto dalla sezione è quello di prendere nota delle norme che andranno a regolare la stesura della documentazione durante il ciclo di vita del software, così da ottenere documenti formalmente concordi.
+\subsubsection{Descrizione}
+La sezione contiene le norme tramite le quali avviene una corretta stesura dei documenti, comprendente verifica e approvazione. I membri del gruppo sono tenuti ad attenersi alla suddetta sezione.
```

Comment:
Cambiare
"... avviene una corretta ..."
in
"... avviene la corretta ..."

Reply:
Fixed